src/layered_config_tree/main.py

```diff
@@ -365,18 +370,31 @@ def get(self, keys: str | list[str], default_value: Any = None) -> Any:
 
         Returns
         -------
-        The value at the key or nested keys. If the key path exists except for the
-        final value, the ``default_value`` is returned.
+        The value at the key or nested keys at the requested layer. ``default_value``
+        is returned if
+        - the key path exists except for the final value.
+        - the value does not exist at the requested layer.
```

Comment:
You should also say what happens if a layer isn't provided and the value doesn't exist at all.

Reply:
Does this change make it clearer?